content/how-to-guides/how-to-use-api-key.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: How to use API key
+---
+
+API keys are alpha-numeric string values distributed to the developer customers to grant access to your API.
+
+# Steps to use API Key
+
+This guide helps you with the steps to use the API key to interact with the Stacks Blockchain API.
```

Comment:
As discussed, let's generalize this document in its entirety, given that this guide applies to all 3 APIs we ship/support. 

We may have to find a way to reuse this guide for all 3 product sections, also.